core/tests/puppeteer-acceptance-tests/spec/translation-admin-tests/provide-translation-rights.spec.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Acceptance Test for translation admins to provide translation
+ * rights from users.
```

Comment:
```suggestion
 * @fileoverview Acceptance test for translation admins to add translation
 * rights to users.
```
Nit: Would change "provide" to "add" everywhere (including test filename) to be consistent with the UI. Similarly, rename "revoke" to "remove".

Reply:
Done